### ISSUE TITLE
Hope speech classifier

### ISSUE THREAD

**Issue by gati** [Data4Democracy/far-right-analysis#15]
"Hope speech" is counter-speech - messages posted in response to hate speech on a social platform. This task is to train a classification model that can identify hope speech. 

One path might be, possibly using the classifier from #14, identify hate speech associated with a hashtag we can use the [Assemble](https://github.com/Data4Democracy/assemble) infrastructure to monitor, or the comments of a Facebook page we can use [collect-social](https://github.com/gati/collect-social) to monitor. Then look at the replies to that hate speech, and identify those that are in opposition to the original message. (Figuring out what's in opposition, or building a training set of counter-speech, is the tricky part). 

Once you've built this, ping Jonathon (@gati) - next step will be to see which hope speech is effective.

**Comment by fx86** [comment]
Is someone working on this ?
If not, I'd like to take this up.

**Comment by gati** [comment]
I don't think so, go for it @fx86! Please let me know if there's anything I can do to help!

**Comment by fx86** [comment]
Have a few more questions. Can we do this on Slack ?

**Comment by gati** [comment]
Sorry for the slow response! Pinged you back on Slack. Just give me a shout
if I can help get you started. :)
